docs/concepts/how-snowpack-works.md

```diff
@@ -6,7 +6,7 @@ description: Snowpack serves your application unbundled during development. Each
 
 ### Summary
 
-**Snowpack is a modern, lightweight build tool for faster web development.** Traditional JavaScript build tools like webpack and Parcel need to rebuild & rebundle entire chunks of your application every time you save a single file. This rebundling step introduces lag between hitting save on your changes and seeing them reflected in the browser.
+**Snowpack is a modern, lightweight build tool for faster web development.** Traditional JavaScript build tools like Webpack and Parcel need to rebuild & rebundle entire chunks of your application every time you save a single file. This rebundling step introduces lag between hitting save on your changes and seeing them reflected in the browser.
```

Comment:
So funnily enough, `webpack` (lowercase) actually is the correct capitalization requested by the core team! They liked how it mirrors the npm install command. You’ll find references to this all over their official website and documentation. So we’ll respect their capitalization.